src/compiler/checker.ts

```diff
@@ -2529,35 +2530,6 @@ module ts {
                     (<GenericType>type).target = <GenericType>type;
                     (<GenericType>type).typeArguments = type.typeParameters;
                 }
-                
-                type.declaredProperties = getNamedMembers(symbol.members);
-                type.declaredCallSignatures = emptyArray;

```

Comment:
This code was slightly different than the interface case (i.e. ```emptyArray``` here instead of ```getSignaturesOfSymbol(symbol.members["__call"]);```)  can this manifest itself in any inappropriate ways?